environment.yml

```diff
@@ -26,9 +26,9 @@ dependencies:
   - six=1.15.0
   - typing_extensions=3.7.4.3
   - pip:
-      - awscli==1.22.17
-      - botocore==1.23.17
-      - boto3==1.20.17
+      - awscli==1.23.7
```

Comment:
Should this be 1.23.3?

Reply:
I was raising to the latest versions. We can pin to the versions of the launch instead.

Reply:
No, latest is fine. Will approve.